server/eclipse-project/src/main/java/us/freeandfair/corla/csv/DominionCVRExportParser.java

```diff
@@ -313,9 +313,18 @@ private void updateContestStructures(final CSVRecord the_line,
         index = index + 1;
         count = count + 1;
       }
-      // get the contest name and "(Vote For=" number
-      String cn = c.substring(0, c.indexOf("(Vote For="));
-      final String vf = c.replace(cn, "").replace("(Vote For=", "").replace(")", "");
+
+      // Find how many votes are allowed
+      String voteForString ="(Vote For=";
+      int votesAllowedIndex = c.indexOf(voteForString);
+      if (votesAllowedIndex == -1)  {
```

Comment:
This can be replaced with StringUtils.containsIgnoreCase(c, voteForString). In any case, the check above is case sensitive. If we get a file where a Vote -> vote, this code will fail.